spec/schema.json

```diff
@@ -387,6 +590,64 @@
         }
       }
     },
+    "Address" : {
```

Comment:
Thought this structure would have been in the spec already, if we are storing ROA in Mongo now.